docs/data/data-grid/editing-legacy/editing.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Data Grid - Editing (legacy)
+---
+
+# Data Grid - Editing (legacy)
+
+<p class="description">The data grid has built-in edit capabilities that you can customize to your needs.</p>
+
+## Cell editing
+
+Cell editing allows editing the value of one cell at a time.
```

Comment:
"Allow" is not generally recommended in this context. See: [Google developer style guide word list—enable](https://developers.google.com/style/word-list#enable). I have also added a note about this in [our style guide on Notion](https://www.notion.so/mui-org/Style-guide-2a957a4168a54d47b14bae026d06a24b) for future reference.

```suggestion
Cell editing lets you edit the value of one cell at a time.
```

All the same, this sentence feels redundant: "Cell editing lets you edit cells." I think it could be cut.